source/core/parameters.cc

```diff
@@ -239,6 +239,33 @@ namespace Parameters
     prm.leave_subsection();
   }
 
+  void
+  PowerLawParameters::declare_parameters(ParameterHandler &prm)
+  {
+    prm.enter_subsection("power-law");
+    {
+      prm.declare_entry("K", "1.0", Patterns::Double(), "Viscosity at rest");
+      prm.declare_entry("n", "0.5", Patterns::Double(), "Power parameter");
+      prm.declare_entry("shear rate min",
+                        "0.001",
```

Comment:
This number is totally random, I think there's no such thing as a default minimum shear rate, it will really depend on K and n.